huxley/www/js/components/ChairAttendanceView.js

```diff
@@ -89,12 +96,16 @@ var ChairAttendanceView = React.createClass({
               </tbody>
             </table>
           </div>
-          <Button
-            color="green"
-            onClick={this._handleSaveAttendance}>
-            Confirm Attendance
-          </Button>
         </form>
+        <div className="foot-bar">
```

Comment:
UI-wise, I'm opposed to having these buttons in a footer; a footer isn't specific to the particular page, so it's confusing to have page-specific buttons there.

If the issue is that user's shouldn't have to scroll to click save/publish, we could just make the table scrollable itself on the page: http://stackoverflow.com/questions/9707397/making-a-div-vertically-scrollable-using-css

That way, the buttons can be always visible, but the table is still scrollable.